internal/store.go

```diff
@@ -472,6 +472,29 @@ func (s *Store[K, V]) Len() int {
 	return total
 }
 
+func (s *Store[K, V]) Size() int {
+	total := 0
+	for _, s := range s.shards {
+		tk := s.mu.RLock()
+		for _, entry := range s.hashmap {
+			total += int(entry.cost.Load())
+		}
+		s.mu.RUnlock(tk)
+	}
+	return total
+}
+
+func (s *Store[K, V]) EstimatedSize() int {
+	total := s.policy.slru.protected.Len() + s.policy.slru.probation.Len()
+	for _, s := range s.shards {
+		tk := s.mu.RLock()
+		total += s.qlen
+		s.mu.RUnlock(tk)
+	}
+	return total
+}
+
+// spread hash before get index
```

Comment:
I remove this comment from main branch yesterday, please make sure your branch is based on latest main branch.

Reply:
I missed it during rebase, removed